README.md

```diff
@@ -7,10 +8,7 @@
 
 Shuttle is an open source, local music player for Android.
 
-Shuttle comes in two flavours: 
-
-- [Shuttle (free)](https://play.google.com/store/apps/details?id=another.music.player)
-- [Shuttle+](https://play.google.com/store/apps/details?id=com.simplecity.amp_pro)
+# Just download the source code and build your own tracker-free version!
```

Comment:
Let's not call it a 'tracker-free' version. Happy with 'analytics free', or some other phrasing .Default Shuttle doesn't contain 'trackers', it contains a library for collecting anonymous crash data and another for collectin anonymous app usage data.